Project/Assets/ML-Agents/Examples/SharedAssets/Scripts/JointDriveController.cs

```diff
@@ -141,6 +141,17 @@ public void SetupBodyPart(Transform t)
                 bp.groundContact.agent = gameObject.GetComponent<Agent>();
             }
 
+            if (bp.joint)
```

Comment:
Does this change modify the behavior of other Agents ?

Reply:
it will prevent a NaN error if a joint component initializes with null or zero JointDrive settings. there is no negative impact on the other ragdoll envs.